Tests/DependencyInjection/FOSRestExtensionTest.php

```diff
@@ -684,4 +684,24 @@ public function testExceptionThrownIfCallbackFilterIsUsed()
     {
         $this->extension->load(array('fos_rest' => array('view' => array('jsonp_handler' => array('callback_filter' => 'foo')))), $this->container);
     }
+
+    public function testParamConverterClassParameterWithFrameworkExtraBundle20()
```

Comment:
was just pondering .. this test isn't really so useful .. it basically just duplicates the logic. what would be ideal is to have a functional test that just tests if the parameter converters actually work .. but for this we would have to add an actual Kernel and do a proper functional test ..